packages/story-editor/src/app/media/utils/useMediaUploadQueue/test/useMediaUploadQueue.js

```diff
@@ -18,26 +18,41 @@
  * External dependencies
  */
 import { renderHook, act } from '@testing-library/react-hooks';
+import { isAnimatedGif } from '@web-stories-wp/media';
 
 /**
  * Internal dependencies
  */
 import useMediaUploadQueue from '..';
 import useFFmpeg from '../../useFFmpeg';
 
+const canTranscodeFile = (file) => {
+  return ['video/mp4'].includes(file.type);
+};
+
+const waitfor100 = async () => {
+  await new Promise((res) => setTimeout(res, 100));
+};
```

Comment:
Why artificially make the tests slower with some sleep? 🤔 

What's it that this achieves? Is there a way to test this without this sleep?

Reply:
It replicates the real functions and replicate the real functions. Also in my testing, without this slow down, the call to `waitForNextUpdate` would skip to UPLOADED, skipping state change like isTranscoding.

Reply:
Does it work with a slower timeout, e.g. 50ms?

Reply:
It does. I changed it to 10ms, which seems to be min speed I can push it.